cmd/kube-apiserver/app/server.go

```diff
@@ -553,6 +559,16 @@ func BuildAuthorizer(s *options.ServerRunOptions, versionedInformers clientgoinf
 	return authorizationConfig.New()
 }
 
+// BuildRequestManager constructs the request manager
+func BuildRequestManager(s *options.ServerRunOptions, extclient clientgoclientset.Interface, versionedInformer clientgoinformers.SharedInformerFactory) utilflowcontrol.Interface {
```

Comment:
as far as i can remember, in previous reviews, yue suggested to abort requestManager concept here as it was split into filter and controller. is that change going to be in another pr?

Reply:
I am not sure I understand your words.  Are you raising an issue about the name, or about how changes are packaged into PRs, or something else?

This PR includes both the filter and the configuration controller.  Particularly after the recent refactorings, there is very little in the filter; almost all the action is in the configuration controller.

Yes, "request manager" is an outdated term.  I plan to submit a PR in the future to replace it.

Reply:
it's about the name. just to make sure that this name is going to be aborted. thanks for answering.

Reply:
OK.
And this name appears only in this PR, so I will fix it in this PR.